mopro-ffi/src/app_config/ios.rs

```diff
@@ -115,27 +124,33 @@ pub fn build() {
         .unwrap()
         .wait()
         .unwrap();
+
+    // The iOS project expects the module map to be named "module.modulemap",
+    // but uniffi-bindgen creates "moproFFI.modulemap" by default,
+    // therefore we need to rename it.
+    rename_module_map_recursively(&bindings_out);
+
     if let Ok(info) = fs::metadata(&bindings_dest) {
         if !info.is_dir() {
             panic!("framework directory exists and is not a directory");
         }
         fs::remove_dir_all(&bindings_dest).expect("Failed to remove framework directory");
     }
+
     fs::rename(&bindings_out, &bindings_dest).expect("Failed to move framework into place");
     // Copy the mopro.swift file to the output directory
-    cleanup_tmp_local(&build_dir_path)
+    cleanup_tmp_local(build_dir_path)
 }
 
-pub fn write_bindings_swift(out_dir: &Path) {
-    if let Ok(info) = fs::metadata(out_dir) {
-        if !info.is_dir() {
-            panic!("out_dir exists and is not a directory");
+fn rename_module_map_recursively(bindings_out: &PathBuf) {
+    for entry in fs::read_dir(bindings_out).expect("Failed to read bindings out directory") {
+        let entry = entry.expect("Failed to read entry");
+        let path = entry.path();
+        if path.is_file() && path.file_name().unwrap() == "moproFFI.modulemap" {
+            let dest_path = path.with_file_name("module.modulemap");
+            fs::rename(&path, &dest_path).expect("Failed to rename module map");
```

Comment:
If we're only looking for 1 file an explicit `break` or `return` might be good here.

Reply:
We're looking for multiple as we're normally building for both physical device and the simulator (each corresponding folder has a `modulemap`). I'll add the comment.